docs/decisions/0044-OTel-semantic-convention.md

```diff
@@ -0,0 +1,326 @@
+﻿---
+# These are optional elements. Feel free to remove any of them.
+status: { proposed }
+contact: { Tao Chen }
+date: { 2024-04-15 }
+deciders: {}
+consulted: { Ben Thomas, Liudmila Molkova, Stephen Toub }
+informed: { Dmytro Struk, Mark Wallace }
+---
+
+# Use standardized vocabulary and specification for observability in Semantic Kernel
+
+## Context and Problem Statement
+
+Observing LLM applications has been a huge ask from customers and the community. This work aims to ensure that SK provides the best developer experience while complying with the industry standards for observability in generative-AI-based applications.
+
+For more information, please refer to this issue: https://github.com/open-telemetry/semantic-conventions/issues/327
+
+### Semantic conventions
+
+The semantic conventions for generative AI are currently in their nascent stage, and as a result, many of the requirements outlined here may undergo changes in the future. Consequently, several features derived from this Architectural Decision Record (ADR) may be considered experimental. It is essential to remain adaptable and responsive to evolving industry standards to ensure the continuous improvement of our system's performance and reliability.
+
+- [Semantic conventions for generative AI](https://github.com/open-telemetry/semantic-conventions/tree/main/docs/gen-ai)
+- [Generic LLM attributes](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/llm.md)
```

Comment:
```suggestion
- [Generic LLM attributes](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/gen-ai.md)
```

the file was recently renamed. Once we have semconv release it could make sense to add a link to the tag, but it's not  possible now. Permalink would be out of date too soon :)